common/dynamicconfig/constants.go

```diff
@@ -2525,4 +2525,10 @@ WorkerActivitiesPerSecond, MaxConcurrentActivityTaskPollers.
 		false,
 		`When set to true, logs all RPC/request errors for the namespace, not just unexpected ones.`,
 	)
+
+	ActivityAPIsEnabled = NewGlobalBoolSetting(
+		"admin.activityAPIsEnabled",
```

Comment:
probably `frontend.activityAPIsEnabled` since it's used on frontend?

Reply:
'admin' probably not the best. I will rename it to 'system'

Reply:
I think it should be `frontend` since it controls behavior on the frontend, not behavior throughout the system. All the other API feature flags are on frontend as well